build.zig.zon

```diff
@@ -1,10 +1,11 @@
 .{
     .name = "zemscripten",
-    .version = "0.2.0-dev",
+    .version = "0.2.1-dev",
```

Comment:
Please don't increment the patch number. 0.2.0 is going to be the next (first proper) release version. If there is some issue with version identifier then we should put a number in the postfix.